diff --git a/node_modules/@twurple/auth/lib/providers/RefreshingAuthProvider.d.ts b/node_modules/@twurple/auth/lib/providers/RefreshingAuthProvider.d.ts
index 9b8895d..47c925a 100644
--- a/node_modules/@twurple/auth/lib/providers/RefreshingAuthProvider.d.ts
+++ b/node_modules/@twurple/auth/lib/providers/RefreshingAuthProvider.d.ts
@@ -44,7 +44,7 @@ export interface RefreshConfig {
 export declare class RefreshingAuthProvider implements AuthProvider {
     private readonly _clientId;
     private readonly _clientSecret;
-    private readonly _userAccessTokens;
+    public readonly _userAccessTokens;
     private readonly _userTokenFetchers;
     private readonly _intentToUserId;
     private readonly _userIdToIntents;
@@ -155,7 +155,7 @@ export declare class RefreshingAuthProvider implements AuthProvider {
      */
     getAppAccessToken(forceNew?: boolean): Promise<AccessToken>;
     private _checkIntermediateUserRemoval;
-    private _callOnRefresh;
+    public _callOnRefresh;
     private _fetchUserToken;
     private _refreshUserTokenWithCallback;
     private _fetchAppToken;
