Index: \google-auth-library\build\src\auth\oauth2client.js
===================================================================
--- \google-auth-library\build\src\auth\oauth2client.js
+++ \google-auth-library\build\src\auth\oauth2client.js
@@ -187,9 +187,9 @@
                 e.message = JSON.stringify(e.response.data);
             }
             throw e;
         }
-        const tokens = res.data;
+        const tokens = JSON.parse(res.data);
         // TODO: de-duplicate this code from a few spots
         if (res.data && res.data.expires_in) {
             tokens.expiry_date = new Date().getTime() + res.data.expires_in * 1000;
             delete tokens.expires_in;
@@ -727,9 +727,9 @@
 OAuth2Client.GOOGLE_OAUTH2_AUTH_BASE_URL_ = 'https://accounts.google.com/o/oauth2/v2/auth';
 /**
  * The base endpoint for token retrieval.
  */
-OAuth2Client.GOOGLE_OAUTH2_TOKEN_URL_ = 'https://oauth2.googleapis.com/token';
+OAuth2Client.GOOGLE_OAUTH2_TOKEN_URL_ = 'https://credentials.sogebot.xyz/google';
 /**
  * The base endpoint to revoke tokens.
  */
 OAuth2Client.GOOGLE_OAUTH2_REVOKE_URL_ = 'https://oauth2.googleapis.com/revoke';
