diff --git a/node_modules/howlongtobeat/dist/main/htmlscraper.js b/node_modules/howlongtobeat/dist/main/htmlscraper.js
index ebf471f..e45fb82 100644
--- a/node_modules/howlongtobeat/dist/main/htmlscraper.js
+++ b/node_modules/howlongtobeat/dist/main/htmlscraper.js
@@ -38,6 +38,11 @@ class HtmlScraper {
                     qs: {
                         page: 1
                     },
+                    headers: {
+                      Referer: 'https://howlongtobeat.com/',
+                      'Content-type': 'application/x-www-form-urlencoded',
+                      'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36',
+                    },
                     form: {
                         'queryString': query,
                         't': 'games',
