--- /home/soge/_WORK/sogeBot/node_modules/twitch-js/types/index.d.ts	2019-12-11 14:01:12.484287785 +0100
+++ patches/index.d.ts	2019-12-11 14:01:02.792335223 +0100
@@ -1465,6 +1465,10 @@
      */
     updateOptions(options: Partial<ChatOptions>): void;
     /**
+     * Timeout user on Twitch
+     */
+    timeout: (maybeChannel: string, timeoutUser: string, time: number, reason?: string) => void;
+    /**
      * Send a raw message to Twitch.
      */
     send: Client['send'];
