Index: \ytsr\lib\utils.js
===================================================================
--- \ytsr\lib\utils.js
+++ \ytsr\lib\utils.js
@@ -17,9 +17,11 @@
 
 exports.parseFilters = json => {
   const pc = json.contents.twoColumnSearchResultsRenderer.primaryContents;
   const wrapper = pc.sectionListRenderer || pc.richGridRenderer;
-  const filterWrapper = (wrapper.subMenu || wrapper.submenu).searchSubMenuRenderer.groups;
+  const filterWrapper = (wrapper.subMenu || wrapper.submenu).searchSubMenuRenderer.groups ||
+    json.header.searchHeaderRenderer.searchFilterButton.buttonRenderer.command.openPopupAction.popup
+      .searchFilterOptionsDialogRenderer.groups || [];
   const parsedGroups = new Map();
   for (const filterGroup of filterWrapper) {
     const singleFilterGroup = new Map();
     singleFilterGroup.active = null;
